app/main/views/frameworks.py

```diff
@@ -68,21 +68,23 @@ def framework_dashboard(framework_slug):
             )
 
     drafts, complete_drafts = get_drafts(data_api_client, framework_slug)
-
     supplier_framework_info = get_supplier_framework_info(data_api_client, framework_slug)
     declaration_status = get_declaration_status_from_info(supplier_framework_info)
-    supplier_is_on_framework = get_supplier_on_framework_from_info(supplier_framework_info)
-
     # Do not show a framework dashboard for earlier G-Cloud iterations
     if declaration_status == 'unstarted' and framework['status'] == 'live':
         abort(404)
 
-    application_made = supplier_is_on_framework or (len(complete_drafts) > 0 and declaration_status == 'complete')
+    supplier_is_on_framework = get_supplier_on_framework_from_info(supplier_framework_info)
+    application_made = complete_drafts and declaration_status == 'complete'
```

Comment:
I'd be slightly careful of changing the meaning of `application_made` here, it means we've got the variable meaning one thing for the entirety of the view, but then a slightly different thing in the template. If the supplier is "on framework" they will have had to make an application, and there should be no danger in them being shown a confidence banner because that's also put behind the condition `framework['status'] == 'open'`.